applications/nrf5340_audio/src/main.c

```diff
@@ -210,7 +210,7 @@ void main(void)
 	ret = streamctrl_start();
 	ERR_CHK(ret);
 
-	ret = audio_datapath_tone_play(1000, 400, 1);
+	ret = audio_datapath_tone_play(500, 440, 0.2);
```

Comment:
Hmm, is frequency and duration swapped here?

Reply:
I guess the goal here is to play 440 Hz (A) for 500 ms?

Reply:
You are correct, will change it right away